File: app/src/main/java/com/ben/bencustomerserver/listener/OnItemClickListener.java
package com.ben.bencustomerserver.listener;

import android.view.View;

/**
 * 条目点击事件
 */
public interface OnItemClickListener {
    /**
     * 条目点击
     * @param view
     * @param position
     */
    void onItemClick(View view, int position);
}

File: lib-bencs/src/main/java/com/ben/bencustomerserver/listener/OnItemClickListener.kt
package com.ben.bencustomerserver.listener

import android.view.View

/**
 * 条目点击事件
 */
interface OnItemClickListener {
    /**
     * 条目点击
     * @param view
     * @param position
     */
    fun onItemClick(view: View?, position: Int)
}